app/src/main/java/com/muratozturk/mova/common/Constants.kt

```diff
@@ -10,6 +10,9 @@ object Constants {
     const val NETWORK_PAGE_SIZE = 10
     const val STARTING_PAGE = 1
 
+    object WebPage{
+        const val PRIVACY_POLICY_URL = "https://github.com/TechnoStudyMobile/Mova#readme"
```

Comment:
No need to put this into "object" class. "const val" can exist outside the class.